submit/controllers/withdraw.py

```diff
@@ -50,10 +50,10 @@ def request_withdrawal(method: str, params: MultiDict, session: Session,
     # Will raise NotFound if there is no such submission.
     submission, submission_events = load_submission(submission_id)
 
-    # The submission must be published for this to be a real JREF submission.
+    # The submission must be published for this to be a withdrawal request.
     if not submission.published:
         alerts.flash_failure(Markup("Submission must first be published. See "
-                                    "<a href='https://arxiv.org/help/jref'>"
+                                    "<a href='https://arxiv.org/help/withdraw'>"
```

Comment:
Let's use ``url_for`` here, should be ``{{ url_for('help_withdraw') }}`` based on the config changes above.

Reply:
Can do. Will that work within the quoted HTML? I wasn't sure if Jinja variables would render in error messages, but I suppose it makes sense that they would.
Our convention so far (IIRC) is that we have hardcoded URLs in our flash/error messages. If we can use the variables, I'll have a covert/backburner task to update those when we make another pass through everything.

Reply:
Actually you might be totally right about that. What I get for doing early morning PRs... Sorry for the misdirect. We should add support for Jinja in alerts in base

Reply:
What you could do, though, is use ``flask.url_for`` directly in code, and then put that in your message